src/Chainweb/Pact/PactService.hs

```diff
@@ -1284,6 +1278,14 @@ transactionsFromPayload plData = do
     toCWTransaction bs = evaluate (force (codecDecode chainwebPayloadCodec $
                                           _transactionBytes bs))
 
+debugResult :: A.ToJSON a => Text -> a -> PactServiceM cas ()
```

Comment:
I'm assuming debugging coinbase and tx outputs won't be a perf problem if `Debug` is not enabled, assuming the logger is lazy?